Source/WebCore/css/CSSFontFeatureValue.cpp

```diff
@@ -44,8 +44,8 @@ String CSSFontFeatureValue::customCSSText(const CSS::SerializationContext& conte
     StringBuilder builder;
     builder.append('"', m_tag[0], m_tag[1], m_tag[2], m_tag[3], '"');
     // Omit the value if it's `1` as `1` is implied by default.
-    if (m_value->resolveAsIntegerIfNotCalculated() != 1)
-        builder.append(' ', m_value->customCSSText(context));
+    if (RefPtr primitiveValue = dynamicDowncast<CSSPrimitiveValue>(m_value); !primitiveValue || primitiveValue->resolveAsIntegerIfNotCalculated() != 1)
```

Comment:
I think it reads better reversed. Might be even better with a `isKnownOne` function.
```suggestion
    if (RefPtr primitiveValue = dynamicDowncast<CSSPrimitiveValue>(m_value); !(primitiveValue && primitiveValue->resolveAsIntegerIfNotCalculated() == 1))
```